### ISSUE TITLE
Scrolling doesn't work after latest Yosemite updates

### ISSUE THREAD

**Issue by lowski** [saitoha/mouseterm-plus#9]
Hi,
It looks like after some recent update (sorry I don't remember which one but I suspect XCode) the plugin doesn't work anymore. I also don't see `Mouse` button in the `Keyboard` section under `Profiles`.

Any chances to fix that?

**Comment by oystersauce8** [comment]
My company did a forced update of yosemite today, and looks like this plugin doesnt work for me too.

**Comment by saitoha** [comment]
@lowski @oystersauce8 mouseterm-plus requires SIMBL.
Have you checked it? Or can you launch EasySIMBL?
https://github.com/norio-nomura/EasySIMBL/releases

**Comment by lowski** [comment]
@saitoha yes I have EasySIMBL installed and mouseterm-plus enabled. It was working before, after recent updates it stopped.

**Comment by oystersauce8** [comment]
of course i have easysimbl installed too. without easysimbl this simply wouldnt work.
anyway I can provide remote desktop access if anyone wants to see this issue in action
in yosemite + terminal + emacs

**Comment by cHoco** [comment]
EasySIMBL doesn't work in the latest Yosemite update due to new security policies. You need to switch to plain old SIMBL.

**Comment by lowski** [comment]
@cHoco thanks for the tip - it works with old SIMBL fine!

**Comment by Congee** [comment]
@cHoco Thx :D

**Comment by cHoco** [comment]
You guys probably want to disable mouseterm mouse functionality if you're using latest Yosemite version, since Terminal added official support for mouse reporting. But you should still use it for all the other awesome features :D
